### DISCUSSION TITLE
CM4S support

### DISCUSSION THREAD

**Discussion by alfonsobozzelli** [monome/norns#1742] [General]
Out of curiosity I've bought a CM4S board, made some checks and it's working (more or less, I've not performed deep test). I don't know if there's any real benefit except some concern about future CM3 boards availability so maybe it's worth a look.
I've added some conditional filters in config.txt (kernel  switch and usb hub enabled) to the stock Norns image (norns231114.img) and performed a regular fresh install, that's all.

```
[pi3]
kernel=kernel-5.10.92-18-g458e2253667a.img

[pi3+]
kernel=kernel-5.10.92-18-g458e2253667a.img

[pi4]
kernel=kernel-5.10.92-18-v7l-g458e2253667a.img

[cm4s]
dtoverlay=dwc2,dr_mode=host

```

**Comment by ambv** [comment]
I can confirm this works for me.

**Comment by hallmar** [comment]
Maybe wishful thinking, but the CM4S does have support for an external wi-fi antenna :^)
A redesign of the bottom plate might be needed though.

**Comment by tehn** [comment]
looking at a CM4S right now, i'm not seeing a wifi transceiver or antenna
connector. do you have a reference?

**Comment by hallmar** [comment]
Ah sorry, I got confused with looking at the CM4, because Google thought I meant CM4 instead of CM4 **S**
